### DISCUSSION TITLE
[Bug Report] Cannot Install

### DISCUSSION THREAD

**Discussion by urliltyger** [stashapp/stash#3341] [Q&A]
**Describe the bug**
I am a Mac User. I have a macbook laptop. macOSSierra. I tried downloading the file multiple times.  [stash-macos-applesilicon](https://github.com/stashapp/stash/releases/download/latest_develop/stash-macos-applesilicon)

It does not work. It gives me this text file full of squiggles. I assume this is coding language. I have no idea what any of it means. 
![Screen Shot 2022-12-30 at 11 35 08 PM](https://user-images.githubusercontent.com/121710346/210129211-9d03af4d-e8b3-4f9d-9777-ac54bd1e5fae.png)

I went to my terminal and typed in ./stash-osx 
All it does is say: No such file or directory
Then I typed in chmod u+x stash-osx
All it does is say: No such file or directory

I don't know any of it means. I don't know what local host 9999. mean either. I put it in my URL and it says error.

**Comment by ALonelyJuicebox** [comment]
This is probably better suited for the Discussions (or discord), I disagree that this is a bug

Are you on a macbook with Apple Silicon (M1 or M2) or do you have an Intel chip? Based on the version of macOS you're running I'd hazard a guess that your macbook is a little older and has an Intel chip. You downloaded the wrong release in that instance
https://github.com/stashapp/stash/releases/download/v0.18.0/stash-macos-intel

**Comment by DogmaDragon** [comment]
If the filename is `stash-macos-applesilicon` command should also use it instead of  `stash-osx`.

https://docs.stashapp.cc/docs/Getting-Started/Installation/Installing-on-macOS/

**Comment by urliltyger** [comment]
> https://github.com/stashapp/stash/releases/download/v0.18.0/stash-macos-intel

Hi thank you for the response. My mac is old. Mid 2012 Model. I downloaded the link you sent me. Same issue. 
![Screen Shot 2023-01-04 at 12 26 19 PM](https://user-images.githubusercontent.com/121710346/210643701-d571efb6-a6ff-459e-9fd8-645bd97b23f9.png)

Is my mac simply too old for Stash app?

**Comment by urliltyger** [comment]
Thank you for reply. 

I downloaded this: https://github.com/stashapp/stash/releases/download/v0.18.0/stash-macos-intel

and typed in  ./stash-macos-intel using terminal, the response from terminal is: No such file or directory
then I typed in chmod u+x stash-macos-intel, the response from terminal is: no such file or directory

**Comment by Flashy78** [comment]
This is not a bug for Github.

You should close this bug and visit the Help channel on Discord to get help specific to you for how to run a file on MacOS.
https://discord.gg/frQ7qBvB3S

**Comment by DogmaDragon** [comment]
> Thank you for reply.
> 
> I downloaded this: https://github.com/stashapp/stash/releases/download/v0.18.0/stash-macos-intel
> 
> and typed in ./stash-macos-intel using terminal, the response from terminal is: No such file or directory then I typed in chmod u+x stash-macos-intel, the response from terminal is: no such file or directory

You need to launch the terminal in the same directory where your downloaded file is or navigate to it manually.
